image.go

```diff
@@ -1,24 +1,29 @@
 package yoti
 
-import "encoding/base64"
+import (
+	"encoding/base64"
+)
 
-//ImageType Image format
+//Deprecated: Will be removed in v3.0.0 - use attribute.ContentType instead. ImageType Image format
 type ImageType int
 
 const (
 	//ImageTypeJpeg JPEG format
 	ImageTypeJpeg ImageType = 1 + iota
 	//ImageTypePng PNG format
 	ImageTypePng
+	//ImageTypeOther Other image formats
+	ImageTypeOther
```

Comment:
I don't think we support any other image formats.
If and when we do, they'll have names.

Reply:
This is there just to have an option for the imageType if it doesn't match either PNG or JPEG. We don't ever want this to be used, since we will introduce a new type, it's there just to cover the "just in case" scenario